common/ledger/util/leveldbhelper/leveldb_provider_test.go

```diff
@@ -191,6 +191,55 @@ func TestBatchedUpdates(t *testing.T) {
 	}
 }
 
+func TestRemove(t *testing.T) {
```

Comment:
Can you please check whether it is possible to use sub-test similar to what we have done here
https://github.com/hyperledger/fabric/blob/da0c6efff04b244ce0963ba4dba82150a1ef7768/common/ledger/util/leveldbhelper/leveldb_provider_test.go#L44-L79

If possible, I would suggest changing this to sub-test as it would improve the readability significantly. 

Nit. `TestRemove` does not match the function that it test.